tools/openms/OpenSwathWorkflow.xml

```diff
@@ -7,12 +7,22 @@
   </macros>
   <expand macro="stdio"/>
   <expand macro="requirements"/>
-  <command>OpenSwathWorkflow
-
--in
-  #for token in $param_in:
-    $token
-  #end for
+  <command>
+#import re
+#set $input_files = []
+#for infile in $param_in:
+  #if str($infile.ext) == 'mzxml':
+    #set $file_extension = '.mzXML'
+  #else:
+    #set $file_extension = '.mzML'
+  #end if
+  #set $input_name = '"' + re.sub('\W','_',$infile.display_name) + $file_extension + '"'
```

Comment:
Do we really need the display name here? I'm a little bit worried about special characters ...